specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/examples/VirtualNetworkGatewayConnectionGetIkeSas.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "api-version": "2020-07-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "virtualNetworkGatewayConnectionName": "vpngwcn1"
```

Comment:
this example is missing an example of the input to the POST call

Reply:
We won't be passing anything in the body. The parameters are directly passed from the headers.